packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -15,16 +27,21 @@ export interface LabelVariables {
 export default (siteVars: SiteVariablesPrepared): LabelVariables => {
   const color = 'rgba(0, 0, 0, 0.6)'
 
+  const colorScheme = extendColorScheme(siteVars.colorScheme, {
+    default: {
+      background: color,
+      foreground: 'rgb(232, 232, 232)',
+    },
+    brand: {
+      background: siteVars.colorScheme.brand.foreground4,
+    },
+  })
+
   return {
-    colorScheme: extendColorScheme(siteVars.colorScheme, {
-      default: {
-        background: color,
-        foreground: 'rgb(232, 232, 232)',
-      },
-      brand: {
-        background: siteVars.colorScheme.brand.foreground4,
-      },
-    }),
+    colorScheme: pickValuesFromColorScheme<LabelColorComponentAreas>(
```

Comment:
no need to specify generic parameter here if for `colorUtils.ts` the following changes will be introduced:

```js
export function pickValuesFromColorScheme<T extends Partial<ComponentAreaName>>(
  colorScheme: ColorSchemeMapping,
  componentAreaSubset: T[],    // <--------------------- use generic arg here
):
```